src/main/scala/it/pagopa/interop/tenantmanagement/model/persistence/TenantPersistentBehavior.scala

```diff
@@ -111,6 +126,8 @@ object TenantPersistentBehavior {
       case TenantDeleted(tenantId)                      => state.deleteTenant(tenantId)
       case SelfcareMappingCreated(selfcareId, tenantId) => state.addSelfcareMapping(selfcareId, tenantId)
       case SelfcareMappingDeleted(selfcareId)           => state.deleteSelfcareMapping(selfcareId)
+      case TenantMailAdded(tenantId, mail, _)           => state.addTenantMail(tenantId, mail)
+      case TenantMailDeleted(tenantId, mailId)          => state.deleteTenantMail(tenantId, mailId)
```

Comment:
having the tenant, we could just
```suggestion
      case TenantMailAdded(tenantId, _, tenant)           => state.addTenantMail(tenantId, tenant)
      case TenantMailDeleted(tenantId, _, tenant)          => state.deleteTenantMail(tenantId, tenant)
```

and in those functions just update the tenant in the state